src/editors/mesh.rs

```diff
@@ -193,6 +200,20 @@ fn edit_mesh(
         mesh.objects.remove(i);
     }
 
+    if let Some(i) = mesh_to_duplicate {
+        let mut duplicated_mesh = mesh.objects[i].clone();
+        let mut subindices = vec![];
+        for object in &mesh.objects {
+            if object.name == duplicated_mesh.name {
+                subindices.push(object.subindex)
+            }
+        }
+
+        duplicated_mesh.subindex = subindices.iter().max().unwrap_or(&0u64) + 1;
```

Comment:
I would do this like the following to avoid creating a vec.
`mesh.objects.iter().filter(|o| o.name == duplicated_mesh.name).map(|o| o.subindex).max().unwrap_or_default()`